.github/workflows/sweep.yml

```diff
@@ -0,0 +1,30 @@
+name: Sweep
+
+on:
+  schedule:
+    - cron: "0 1 * * *"
```

Comment:
what time zone does it uses?
me and @mhoffm-aiven agreed that it should do that at 3.00 UTC

Reply:
its UTC

Reply:
by default docker Ubuntu image uses UTC, so we are good